src/OrchardCore.Modules/OrchardCore.AuditTrail/Navigation/AuditTrailSettingsAdminMenu.cs

```diff
@@ -26,7 +26,7 @@ public Task BuildNavigationAsync(string name, NavigationBuilder builder)
                  .Add(S["Configuration"], configuration => configuration
                      .Add(S["Settings"], settings => settings
                         .Add(S["Audit Trail"], S["Audit Trail"].PrefixPosition(), settings => settings
-                            .AddClass("audittrail").Id("audittrail")
+                            .AddClass("audittrail").Id("audittrailSettings")
```

Comment:
Where is the other "audittrail" ID coming from?

Reply:
First [instance](https://github.com/OrchardCMS/OrchardCore/blob/ef1e17bbe573a7c00c1ec355c47a334b9e94addc/src/OrchardCore.Modules/OrchardCore.AuditTrail/Navigation/AuditTrailAdminMenu.cs#L27) and [second ](https://github.com/OrchardCMS/OrchardCore/blob/ef1e17bbe573a7c00c1ec355c47a334b9e94addc/src/OrchardCore.Modules/OrchardCore.AuditTrail/Navigation/AuditTrailSettingsAdminMenu.cs#L29).